datafusion/core/tests/sqllogictests/src/main.rs

```diff
@@ -109,7 +109,7 @@ pub async fn main() -> Result<()> {
             info!("Using complete mode to complete {}", path.display());
             let col_separator = " ";
             let validator = default_validator;
-            update_test_file(path, runner, col_separator, validator)
+            update_test_file(path, &mut runner, col_separator, validator)
```

Comment:
needed due to https://github.com/risinglightdb/sqllogictest-rs/commit/879204b9b2dd5a81db4a9781c2e6c68d0713f5d3

Reply:
Yes, I changed it, lol